packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -120,25 +120,59 @@ abstract class FocusTraversalPolicy with Diagnosticable {
   /// A const constructor so subclasses can be const.
   const FocusTraversalPolicy();
 
-  /// Returns the node that should receive focus if there is no current focus
-  /// in the nearest [FocusScopeNode] that `currentNode` belongs to.
+  /// Returns the node that should receive focus if focus is traversing
+  /// forwards, and there is no current focus.
   ///
-  /// This is used by [next]/[previous]/[inDirection] to determine which node to
-  /// focus if they are called when no node is currently focused.
+  /// The node returned is the node that should receive focus if focus is
+  /// traversing forwards (i.e. with [next]), and there is no current focus in
+  /// the nearest [FocusScopeNode] that `currentNode` belongs to.
   ///
   /// The `currentNode` argument must not be null.
   ///
   /// The default implementation returns the [FocusScopeNode.focusedChild], if
   /// set, on the nearest scope of the `currentNode`, otherwise, returns the
   /// first node from [sortDescendants], or the given `currentNode` if there are
   /// no descendants.
-  FocusNode findFirstFocus(FocusNode currentNode) {
+  ///
+  /// See also:
+  ///
+  ///  * [next], the function that is called to move the focus to the next node.
+  ///  * [DirectionalFocusTraversalPolicyMixin.findFirstFocusInDirection], a
+  ///    function that finds the first focusable widget in a particular direction.
+  FocusNode findFirstFocus(FocusNode currentNode) => _findFirstFocus(currentNode);
+
+  /// Returns the node that should receive focus if focus is traversing
+  /// backwards, and there is no current focus.
+  ///
+  /// The node returned is the one that should receive focus if focus is
+  /// traversing backwards (i.e. with [previous]), and there is no current focus
+  /// in the nearest [FocusScopeNode] that `currentNode` belongs to.
+  ///
+  /// The `currentNode` argument must not be null.
+  ///
+  /// The default implementation returns the [FocusScopeNode.focusedChild], if
+  /// set, on the nearest scope of the `currentNode`, otherwise, returns the
+  /// last node from [sortDescendants], or the given `currentNode` if there are
+  /// no descendants.
+  ///
+  /// See also:
+  ///
+  ///  * [previous], the function that is called to move the focus to the next node.
+  ///  * [DirectionalFocusTraversalPolicyMixin.findFirstFocusInDirection], a
+  ///    function that finds the first focusable widget in a particular direction.
+  FocusNode findLastFocus(FocusNode currentNode) => _findFirstFocus(currentNode, findLast: true);
+
+  FocusNode _findFirstFocus(FocusNode currentNode, {bool findLast = false}) {
```

Comment:
Wait, to find the last focus node we call `_findFirstFocus`? :smirk:

Joking aside, perhaps `_findStartFocus` with a `fromEnd` or `backwards` boolean parameter?

Reply:
Yeah, fair point.  Changed to `_findInitialFocus` and `fromEnd`.